Protocol/ErrorMessages.xml

```diff
@@ -2693,7 +2693,7 @@
 								<HasCodeFix>True</HasCodeFix>
 								<HowToFix><![CDATA[]]></HowToFix>
 								<ExampleCode><![CDATA[]]></ExampleCode>
-								<Details><![CDATA[]]></Details>
+								<Details><![CDATA[The minimum required DataMiner version for this connector is lower than the minimum DataMiner version supported by Skyline.[NewLine]For more info on the minimum DataMiner version supported by Skyline, see https://docs.dataminer.services/user-guide/Reference/Software_support_life_cycles.html.]]></Details>
```

Comment:
I would prefer that an aka link is being used, so that if the location on the docs is ever changed, it's a lot easier to update the aka link then release a new version.

Reply:
adapted to https://aka.dataminer.services/SoftwareSupport-LifeCycle